test/uk/gov/hmrc/apiplatformjobs/connectors/ApplicationCommandConnectorSpec.scala

```diff
@@ -73,7 +73,7 @@ class ApplicationCommandConnectorSpec extends AsyncHmrcSpec with ResponseUtils w
       val anAdminEmail          = "admin@example.com".toLaxEmail
       val developerCollaborator = Collaborators.Developer(UserId.random, "dev@example.com".toLaxEmail)
       val jsonText              =
-        s"""{"command":{"actor":{"email":"${anAdminEmail.text}","actorType":"COLLABORATOR"},"collaborator":{"userId":"${developerCollaborator.userId.value}","emailAddress":"dev@example.com","role":"DEVELOPER"},"timestamp":"2020-01-01T12:00:00Z","updateType":"removeCollaborator"},"verifiedCollaboratorsToNotify":["admin@example.com"]}"""
+        s"""{"command":{"actor":{"email":"${anAdminEmail.text}","actorType":"COLLABORATOR"},"collaborator":{"userId":"${developerCollaborator.userId.value}","emailAddress":"dev@example.com","role":"DEVELOPER"},"timestamp":"2020-01-01T12:00:00.000Z","updateType":"removeCollaborator"},"verifiedCollaboratorsToNotify":["admin@example.com"]}"""
```

Comment:
Has something changed formats.  Just wondering if it's because the appDomainVersion was so old.  Or if we're now got some issue somewhere.

Reply:
I honestly don't know.  I think we should be okay but.....  Go with it.  We'll catch any unlikely issues in QA I reckon.